src/test/kotlin/org/rust/lang/core/type/RsPatternMatchingTest.kt

```diff
@@ -422,27 +420,27 @@ class RsPatternMatchingTest : RsTypificationTestBase() {
         } //^ &i32
     """)
 
-    fun `test match ergonomics enum mut`() = expect<IllegalStateException> {
-        testExpr("""
+    fun `test match ergonomics enum mut`() = testExpr("""
         enum E { A(i32) }
         fn main() {
             let E::A(mut a) = &E::A(1);
             a;
         } //^ i32
     """)
-    }
 
+    // TODO
+    // TODO
+    // TODO
+    // TODO
```

Comment:
4 `TODO`s? why?

Reply:
Oh, I added them not to miss it before commit, but I missed it anyway :(